File: Duplicati/Server/Duplicati.Server.Serialization/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Duplicati.GUI.Serialization
{
    public class Schedule
    {
        public long ID { get; private set; }
        public string Name { get; set; }

        public string Path { get; set; }
        public DateTime When { get; set; }
        public string Repeat { get; set; }
        public DateTime NextScheduledTime { get; set; }
        public DayOfWeek[] AllowedWeekdays { get; set; }
    }
}

File: Duplicati/Server/webroot/Duplicati/model/Schedule.js
Ext.define('Duplicati.model.Schedule', {
    extend: 'Ext.data.Model',

    associations: [{ 
    	name: 'task',
    	type: 'hasOne', 
    	model: 'Duplicati.model.Task', 
    	primaryKey: 'ID', 
    	foreignKey: 'ScheduleID' 
    }],

    fields: [
        {
            name: 'NextScheduledTime'
        },
        {
            name: 'AllowedWeekdays'
        },
        {
            name: 'ID'
        },
        {
            name: 'Name'
        },
        {
            name: 'Path'
        },
        {
            name: 'When'
        },
        {
            name: 'Repeat'
        },
        {
            name: 'Weekdays'
        },
        {
            name: 'Metadata'
        }
    ]
});